packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/index.js

```diff
@@ -33,6 +34,11 @@ const ApiTokenListView = () => {
   } = useRBAC(adminPermissions.settings['api-tokens']);
   const { push } = useHistory();
   const { trackUsage } = useTracking();
+  const { startSection } = useGuidedTour();
+
+  useEffect(() => {
+    startSection('apiTokens');
+  });
```

Comment:
Are we sure we want the effect to be ran on every render?

Reply:
no it shouldn't good catch, thanks!